### PR TITLE
Fix time and frequency of SMIR meetings

### PR DESCRIPTION
**Pull request by spastorino**
r? @davidtwco 

cc @oli-obk @celinval

### CODE REVIEW

**Comment by celinval** [comment]
Awesome! I just don't know how to verify if the week that it will skip is in sync with ours.

**Comment by davidtwco** [comment]
> Awesome! I just don't know how to verify if the week that it will skip is in sync with ours.

It'll skip the week after the first week and every other week thereafter (so 19th Jan, then 2nd Feb, etc) - if that's right then we can merge this.

**Review by davidtwco — APPROVED** [review]

File: project-stable-mir.toml [hunk 1]
```diff
@@ -7,9 +7,9 @@ title = "Stable MIR Weekly Meeting"
 description = "Weekly meeting to discuss, plan and implement Stable MIR"
 location = "#project-stable-mir on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/320896-project-stable-mir)"
 last_modified_on = "2024-01-05T16:04:00.00Z"
```
Comment (davidtwco) [inline project-stable-mir.toml:9]: actually, make sure to bump this as well

**Comment by spastorino** [comment]
Comment addressed and the schedule is right ... following one is on Feb 2nd

**Pull request merged**